server/game/cards/01-Core/HidaKisada.js

```diff
@@ -18,13 +19,17 @@ class HidaKisada extends DrawCard {
     }
 
     onInitiateAbilityEffectsWouldInterrupt(event) {
-        if(!this.firstActionEvent && this.game.isDuringConflict() && event.context.ability.abilityType === 'action' && !event.context.ability.cannotBeCancelled && event.context.player !== this.controller) {
-            this.firstActionEvent = event;
+        if(!this.firstActionEvent) {
```

Comment:
This shouldn't be here

Reply:
This still shouldn't be here....

Reply:
yeah you are right, problem was the unchanged onConflictDeclared code.